frontend/src/tests/utils.js

```diff
@@ -174,8 +175,8 @@ function _ensurePath(path) {
  * @param {string} url The api un-prefixed url route (e.g. "/sessions")
  * @returns
  */
-export async function apiGET(url) {
-    url = API_URL + _ensurePath(url);
+export async function apiGET(url, omitPrefix = false) {
+    url = omitPrefix ? URL + _ensurePath(url) : API_URL + _ensurePath(url);
```

Comment:
I think this have been changed on master too, did you rebase the latest one?

Reply:
From what I see, this is the latest on master as well. What change are you referring to?

Reply:
(please see above :) )